.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto
+*.less text eol=lf
```

Comment:
It's not a bad thing. It will prevent Windows users from putting wrong EOL when they will contribute.